docs/product-assembly_designing-and-building-cases/src/main/asciidoc/case-management-dynamic-service-task-API-proc.adoc

```diff
@@ -0,0 +1,58 @@
+[id='case-management-dynamic-service-task-API-proc']
+= Creating a dynamic service task using the REST API
+
+Service tasks are generally less complex than user tasks, though they might need more data to be provided to execute properly. Service tasks require following information:
```

Comment:
although they...lose 'to be provided'
Server tasks require the following information